transactions/test/add_type_to_factory.cdc

```diff
@@ -3,13 +3,14 @@ import "NFTProviderFactory"
 
 import "NonFungibleToken"
 
-transaction {
+transaction(type: Type) {
     prepare(account: AuthAccount) {
-        let managerRef = account.borrow<&CapabilityFactory.Manager>(from: CapabilityFactory.StoragePath)
-            ?? panic("CapabilityFactory Manager not found")
+        let managerRef = account.borrow<&CapabilityFactory.Manager>(
+            from: CapabilityFactory.StoragePath
+        ) ?? panic("CapabilityFactory Manager not found")
     
         let nftProviderFactory = NFTProviderFactory.Factory()
 
-        managerRef.addFactory(Type<&{NonFungibleToken.Receiver}>(), nftProviderFactory)
+        managerRef.addFactory(type, nftProviderFactory)
```

Comment:
Since we're dealing with NFTProviderFactory, suggest renaming file to `add_type_for_nft_provider_factory` or similar distinction

Reply:
Ah yes, good idea :+1: 
Updated in https://github.com/onflow/hybrid-custody/pull/161/commits/5946bbf1c1d30ece1b36926948687069969aac89